source/edge-server/connect.txt

```diff
@@ -170,58 +203,122 @@ modifications you can make to the :github:`Swift template app
+      After the next time the Edge Server instance syncs with your App Services 
+      App, you can see synced changes reflected in the linked Atlas collection.
 
       .. important::
 
          If you have previously connected directly from the client to your 
          App Services App with Device Sync, you may need to clear state on the 
          simulator or device. Resetting your cache enables your client to 
-         connect to the Edge Server.
+         connect to the Edge Server instance.
 
          In a console app, delete the ``mongodb-realm`` directory. If you're
          using an iOS or Android simulator or device, uninstall and reinstall
          the app.
 
 .. _edge-server-wireprotocol-connect:
 
-Connect to the Edge Server with a MongoDB URI
----------------------------------------------
+Connect to the Edge Server with MongoDB Drivers and Tools
+---------------------------------------------------------
 
 You can connect to the Edge Server using standard MongoDB Drivers and tools.
 Clients use a specialized MongoDB URI connection string to connect to 
 Edge Server and send requests.
 
-The wire protocol server is on by default. You can connect to it via the 
-host machine using this MongoDB URI:
+The Wire Protocol server accepts these incoming connections, and is on by 
+default. You can connect to it using the host machine using this MongoDB URI:
 
 .. code:: console
 
-   mongodb://EDGE_SERVER_HOSTNAME:27021
+   mongodb://<username>:<password>@<hostname>:<port>?authMechanism=PLAIN&appName=::<auth-type>
+
+.. list-table::
+   :header-rows: 1
+
+   * - Element
+     - Description
+
+   * - ``<username>``
+     - The username of the Atlas user. For an 
+       :ref:`email/password user <email-password-authentication>`, this 
+       might look similar to ``first.last@example.com``.
+
+   * - ``<password>``
+     - The password of the Atlas user. This might look similar to 
+       ``myPassword123456``.
 
-The ``EDGE_SERVER_HOSTNAME`` is the public DNS of your Edge Server host. 
-For more information, refer to :ref:`edge-server-install-and-configure`.
+   * - ``<hostname>``
+     - The public DNS of your Edge Server host. While in local development, 
+       this might be ``localhost``. Otherwise, it might look similar to
+       ``ec2-13-58-70-88.us-east-2.compute.amazonaws.com``. For more 
+       information, refer to :ref:`edge-server-install-and-configure`.
 
-You must make port 27021 accessible to the client through port mapping or
-firewall rules.
+   * - ``<port>``
+     - The Edge Server Wire Protocol listen port. The default port is ``27021``.
+       For more details, refer to :ref:`edge-server-install-and-configure`.
+
+   * - ``authMechanism=PLAIN``
+     - The auth mechanism. Currently, the only supported value is ``PLAIN``.
+
+   * - ``appName``
+     - Optional string ``appName``. You may omit the value and/or key.
+
+   * - ``<auth-type>``
```

Comment:
i think this is confusing in that this is not needed for local-userpass or anonymous. maybe we can just drop it for now and add it back when we support other providers?

Reply:
Since neither `appName` nor `<auth-type>` is required currently, I've just removed them both to simplify this. Happy to add them back when we require them.